src/reducers/passphrase.js

```diff
@@ -70,10 +76,12 @@ const errors = (state = null, action) => {
     case UPDATE_PASSPHRASE_2FA_1:
     case UPDATE_PASSPHRASE_SUCCESS:
     case UPDATE_PASSPHRASE_2FA_2_SUCCESS:
+    case UPDATE_HINT_SUCCESS:
       return null
     case UPDATE_PASSPHRASE_2FA_1_FAILURE:
     case UPDATE_PASSPHRASE_2FA_2_FAILURE:
     case UPDATE_PASSPHRASE_FAILURE:
+    case UPDATE_HINT_FAILURE:
```

Comment:
This code seems too complex from what it is doing. Everytime we add something in the form we have to add stuff here :/ IMHO, redux should not be used here as the state is not shared between a lot of components or between components that do not share a common parent. This is not in the scope of the PR.